ecleankernel/bootloader/grub.py

```diff
@@ -11,7 +12,10 @@
 class GRUB(LILO):
     name = 'grub'
     kernel_re = r'^\s*(kernel|module)\s*(\([^)]+\))?(?P<path>\S+)'
-    def_path = ('/boot/grub/menu.lst', '/boot/grub/grub.conf')
+    def_path = (os.getenv("GRUB_CFG"),
```

Comment:
Sorry for missing it.

```suggestion
    def_path = (os.environ.get("GRUB_CFG"),
```

Reply:
What's the difference between the two?

Reply:
Yeah, `os.environ` seems more Pythonic to me.